CHANGELOG.md

```diff
@@ -4,6 +4,52 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.1.0
+
+*December 13, 2023*
+
+This is the official v1.1.0 release of NGINX Gateway Fabric.
```

Comment:
I wonder if this sentence adds anything - it should be clear from the title above that it is Release 1.1.0 and that it is NGINX Gateway Fabric (since this readme in that repo).

Reply:
My bad; I copied this from the 1.0 release entry, thinking it was just the standard.

Reply:
Looking at the other changelog entries, this should be a summary of what the release includes. @mpstefan any suggestions?

Reply:
Something like...

"This release updates NGINX Gateway Fabric to support version 1.0 of the Gateway API in addition to bug fixes and documentation updates."

Reply:
Updated